Cargo.toml

```diff
@@ -7,9 +7,11 @@ authors = ["The Rust-gnome Project Developers"]
 libc = "0.1"
 glib = "^0"
 gdk = "^0"
-gtk = "^0"
 cairo-rs = "^0"
 
+[dependencies.gtk]
+features = ["gtk_3_10", "cairo_1_12"]
```

Comment:
While I would prefer to have `add_from_file`, I think that would be a good thing to have an example using GTK 3.10, so next contributors can have something they can copy for other examples requiring 3.10.